REQUIREMENTS.txt

```diff
@@ -1,4 +1,5 @@
 # You should not edit this file directly.  Instead, you should edit one of the following files (REQUIREMENTS.txt.in) and run make REQUIREMENTS.txt
+anndata==0.6.10
```

Comment:
I still think we should _not_ include these dependencies as exporters are not in the realm of core functionality.  One could easily imagine this getting out of hand.

Reply:
Cool, what's the pattern that you'd prefer to see here? An attempt to import where an error is thrown if the dependency is not installed that instructs the user to pip install the thing?

Reply:
I think starfish-full should install everything used by the codebase and the notebooks.

Reply:
> Cool, what's the pattern that you'd prefer to see here? An attempt to import where an error is thrown if the dependency is not installed that instructs the user to pip install the thing?

Yes, that's correct.